src/main/resources/main_fi.properties

```diff
@@ -0,0 +1,48 @@
+# Main text translations
+main.startPrompt=Koemnnot: arvioi salasana (e); luo salasana (g); lopeta (q)
+main.enterCommand=Ole hyv\u00E4 ja sy\u00F6t\u00E4 komentosi:
+main.estimatePrompt=Ole hyv\u00E4 ja sy\u00F6t\u00E4 arvioitavan salasanan:
+main.generatorType=Mink\u00E4 tyyppisen salasanan haluat luoda? Satunnainen (r), Tunnuslause (p):
+main.randomType=Alfa (1), Alfanumeerinen (2), Alfanumeerinen & Symboleja (3), Numeerinen (4):
+main.lengthPrompt=Montako merkki\u00E4:
+main.delimiterPrompt=Mit\u00E4 erotinmerkki\u00E4 haluat k\u00E4ytt\u00E4\u00E4:
+main.wordsPrompt=Montako sanaa haluat tunnuslauseesi sis\u00E4lt\u00E4v\u00E4n:
+main.quitPrompt=Kiitos nbvcxz:n k\u00E4yt\u00F6st\u00E4!
+main.timeToCalculate=Arviointi aika:
+main.password=Salasana:
+main.feedback.warning=Varoitus:
+main.feedback.suggestion=Ehdotus:
+main.feedback.minimumMet=Salasanasi t\u00E4ytt\u00E4\u00E4 minimi vahvuus vaatimuksia.
+main.feedback.insecure=Salasanasi ei t\u00E4yt\u00E4 minimi vahvuus vaatimuksia.
+main.entropy=Entropia:
+main.timeToCrack=Murtamis aika:
```

Comment:
Maybe "Murtamiseen vaadittava aika:" as it follows with an estimate of how long it would take to crack the password, not the actual time it required to crack the password.

Reply:
Yes, I like that.

Reply:
amended